src/Microsoft.CodeAnalysis.ResxSourceGenerator/Microsoft.CodeAnalysis.ResxSourceGenerator/AbstractResxGenerator.cs

```diff
@@ -115,6 +115,12 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
                         publicResource = false;
                     }
 
+                    var noWarn = Array.Empty<string>();
+                    if (options.TryGetValue("build_metadata.AdditionalFiles.NoWarn", out var noWarnText))
```

Comment:
I don't think this will work without `CompilerVisibleItemMetadata`

Reply:
That's right. I hadn't seen these MetaData. 
It's been corrected.